README.md

```diff
@@ -480,6 +480,55 @@ options are the same as [already documented](#getting-started).
+explicit about possible failures, allowing for review of the logs and correction, than to silently ignore the failure
+and possibly allow untrusted code into the repository.
+
+There are several reasons a dependency file may fail the filtering process and each failure will be included in the logs
+as a warning. The file may not exist or it may exist, but only as an empty file. The file may fail to be parsed by
+Phylum. The dependency files can be manifests or lockfiles and they can either be provided explicitly or automatically
+detected when not provided. Sometimes the automatic detection will misattribute a file as a manifest or assign the wrong
+lockfile type. As detailed in the ["Supported Dependency Files"](#supported-dependency-files) section, the
+recommendation for this situation is to specify the path and lockfile type explicitly in a `.phylum_project` file at
+the root of the project repository.
+
+### Why does analysis fail for PRs from forked repositories?
+
+Another reason why Phylum reports
+[failing status checks](#why-does-phylum-report-a-failing-status-check-if-it-shows-a-successful-analysis-comment) is for
+`pull_request_target` events where manifests are provided. Using `pull_request_target` events for forked repositories
+has security implications if done improperly. Attackers may be able to obtain repository write permissions or steal
+repository secrets. A more comprehensive enumeration of the risks can be found here:
+
+* GitHub Security Lab: ["Preventing pwn requests"][gh_pwn]
+* GitGuardian: ["GitHub Actions Security Best Practices"][gha_security]
+
+This GitHub action disables lockfile generation to prevent arbitrary code execution in an untrusted context, like PRs
+from forks. This means that provided manifests are unable to be parsed by Phylum since parsing first requires generating
+a lockfile from the manifest. A unique error code and warning message is provided so as to better signal the
+implication: the resolved dependencies from the manifest have NOT been analyzed by Phylum. Care should be taken to
+inspect changes manually before allowing a manifest to be used in a trusted context.
+
+One way to avoid this situation and enable automatic Phylum analysis is to add a lockfile instead of or along with the
+manifest. This is recommended even for libraries. A great case for why
+[lockfiles should be committed on **all** projects][lockfiles_always] was made by the folks who built the Yarn package
+manager for the npm ecosystem and the same advice applies to other ecosystems. Adding the lockfile is good, but it will
+need to be paired with additional workflow logic that requires it to be updated and current with the corresponding
+manifest as part of the PR submission. Otherwise, it will be possible to supply a valid and benign lockfile while
+updating the manifest with malicious entries (perhaps transitively) to be included the next time the lockfile is
+refreshed.
```

Comment:
This suggestion makes no sense... Adding lockfiles does not help this situation in any way precisely because the lockfiles require additional workflow logic to ensure that they are updated. Those additional workflows are not safe to run from `pull_request_target` workflows.

Additionally, the blog post from yarn is 7 years old, not relevant to this discussion, and linking to it might be mistaken as endorsement for their tool... All reasons that we should not link to it.

The entire paragraph should be removed.

Reply:
This is a tough one...a real catch-22. It has been removed.